### ISSUE TITLE
from "bulle" to note number for <app> in right margin of display

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/tfd-nusantara-philology#14]
@ajaniak:

at the moment, display of right margins has a "bulle" for each "app", as in screenshot:

<img width="1611" alt="Capture d’écran 2021-03-11 à 05 06 22" src="https://user-images.githubusercontent.com/835603/110734343-e6a76e80-8227-11eb-8f28-0faafedf65a8.png">

it is possible to repace the "bulles" by the note numbers that presently become visible only when you mouse over the "bulle" (or over the corresponding red part of the text)?

incidentally, it seems the width of the text block is again responsive and not of fixes width. at the moment, I forget why at one time we (or I) thought a fixed width was better. can you explain your considerations pro and con?

**Comment by ajaniak** [comment]
I have changed the bulles. 

Yes, because something I need the full width length for dev purposes. I have reverted it though.

**Comment by arlogriffiths** [comment]
Thanks!

This is how it looks now:

<img width="1337" alt="Capture d’écran 2021-03-11 à 17 08 22" src="https://user-images.githubusercontent.com/835603/110817296-72eb7d00-828c-11eb-8b15-d12eb7d99741.png">

- can the problem with apparently overlapping numbers (only in the `<lg>` context) be solved?
- I was thinking the note numbers could be displayed with the same reddish color as is use to mark the `<lem>` within the edition 
- can we have a fixed minimal distance of ca. 1 cm between leftmost note number and right extremity of the text?

**Comment by ajaniak** [comment]
- the overlapping have been resolved and is already applied on the CSS. Are you up to date with the transformation? 
- for the red, do you mean like in the following screenshot? 
<img width="1024" alt="Capture d’écran 2021-03-11 à 17 21 19" src="https://user-images.githubusercontent.com/33328906/110819281-4fc1cd00-828e-11eb-94de-87063c78c68b.png">

- I still have to restructure all the right notes. But I need to have the final encoding to do so, since I will have to code some javascript and since it will take me some time, I would like to do it once.

**Comment by arlogriffiths** [comment]
yes, now the overlapping is solved. it's starting to look really nice! we'll patiently await the moment that you will undertake that restructuring. shall we close this issue now?

**Comment by ajaniak** [comment]
If the rest is solved, then I guess we can!

**Comment by arlogriffiths** [comment]
Reopening this just as reminder for when you revise the javascript code that it would be nice to have all notes numbers in the right margin aligned with the top level of the text's structural hierarchy. The current display, which has the notes for lowel levels jump to the left, is shown in the screenshot:
<img width="1161" alt="Capture d’écran 2021-03-20 à 09 35 49" src="https://user-images.githubusercontent.com/835603/111864249-1eb55c80-8960-11eb-8fb4-c15b3185b81b.png">
